mpp/shmemx.h4.in

```diff
@@ -63,13 +63,13 @@ SHMEM_CXX_DEFINE_FOR_SYNC(`SHMEM_CXX_WAIT_UNTIL_ALL')
 
```

Comment:
Should `wait_until_all` also be a `return` statement?

Reply:
Ah, nevermind.  It's `void` return type, so don't need it?

Reply:
Nope, return type is `void`.